server.js

```diff
@@ -3,11 +3,12 @@ const express = require('express');
 const app = express();
 const bodyParser = require('body-parser');
 const mongoose = require('mongoose');
-
+const cors = require('cors');
```

Comment:
Ohhh sorry about that issue! You warned me about CORS and I just completely forgot to put it in there as the finishing touch. MY B, looking good

Reply:
No worries! It gets me every time!

Reply:
I also recall you mentioning this but had no idea what a "CORS" was at the time.